lace/Cargo.toml

```diff
@@ -35,16 +35,16 @@ name = "lace"
 path = "bin/main.rs"
 
 [dependencies]
-lace_cc = { path = "lace_cc", version = "0.1.3" }
+lace_cc = { path = "lace_cc", version = "0.1.4" }
 lace_utils = { path = "lace_utils", version = "0.1.2" }
-lace_stats = { path = "lace_stats", version = "0.1.2" }
-lace_codebook = { path = "lace_codebook", version = "0.1.3" }
+lace_stats = { path = "lace_stats", version = "0.1.3" }
+lace_codebook = { path = "lace_codebook", version = "0.1.4" }
 lace_geweke = { path = "lace_geweke", version = "0.1.2" }
-lace_consts = { path = "lace_consts", version = "0.1.2" }
+lace_consts = { path = "lace_consts", version = "0.1.4" }
 lace_data = { path = "lace_data", version = "0.1.2" }
-lace_metadata = { path = "lace_metadata", version = "0.1.3" }
-dirs = "4"
-itertools = "0.10.3"
+lace_metadata = { path = "lace_metadata", version = "0.1.4" }
```

Comment:
I notice that `lace_data` version used was not changed, despite the version of `lace_data` being bumped. Is that deliberate?